packages/mdc-data-table/adapter.ts

```diff
@@ -32,17 +32,79 @@
 import {MDCDataTableRowSelectionChangedEventDetail} from './types';
 
 export interface MDCDataTableAdapter {
+  /**
+   * Adds a class name to row element at given row index excluding header row.
+   *
+   * @param rowIndex Index of row element excluding header row.
+   * @param cssClasses CSS Class string to add.
+   */
   addClassAtRowIndex(rowIndex: number, cssClasses: string): void;
+
+  /**
+   * Returns attribute value of row element at given row index excluding header row.
+   *
+   * @param rowIndex Index of row element excluding header row.
+   * @param attr Attribute string of row element.
+   */
   getAttributeAtRowIndex(rowIndex: number, attr: string): void;
```

Comment:
I think this returns a `string`, not void. Also add an `@return...`

Reply:
Removed this adapter since unused.